irc/client.py

```diff
@@ -866,7 +870,11 @@ def motd(self, server=""):
     def names(self, channels=None):
         """Send a NAMES command."""
         tmpl = "NAMES {channels}" if channels else "NAMES"
-        channels = ','.join(always_iterable(channels))
+        if channels != None:
+            if isinstance(channels, str) or isintance(channels,
+                    basestring):
```

Comment:
basestring is a NameError on Python 3.

Plus, here you see the value of `always_iterable`, providing an elegant, tested replacement for 4-5 lines of code. It's a pattern I've encountered hundreds of times in Python, and others share the sentiment. I'd like to continue to see the use of it.